src/components/uswds/Alert.tsx

```diff
@@ -43,10 +43,7 @@ export function Alert({
   const Heading = headingLevel;
 
   let role = 'region';
-  if (type === 'success') {
-    role = 'status';
-  }
-  if (type === 'error' || type === 'emergency') {
+  if (type === 'error' || type === 'emergency' || type === 'success') {
     role = 'alert';
```

Comment:
Eleni! Thank you _very_ much for pulling all this together. (And for all the research in #531!)

I’m curious if this change might potentially be trading one set of usability issues for another? At least, I was under the impression that `role="alert"`/`aria-live="assertive"` were to be [reserved for _urgent_ messages that need a timely response](https://www.sarasoueidan.com/blog/accessible-notifications-with-aria-live-regions-part-1/#1.-using-the-aria-live-attribute), as they’ll interrupt any other message currently being read to the user by their AT. I’m worried that could be differently disorienting than what we’ve already got.

Dunno; what do you think?

Reply:
That's a good point and I agree that this seems odd. In our use cases, success messages were not being read when they were given a role of `status`, which to me seemed even more disorienting for AT users. But they were read consistently when upgraded to role of `alert`.

Reply:
I'm up for fiddling with the `/prototype/alert` page more to see if we can still get `status` alerts to work more consistently. Adding and removing the status alert from the DOM gave no SR announcements. I tried keeping a status alert in the DOM while updating the text inside of it, but the SR would only read it the first time it changed.

Reply:
I considered that, but ultimately concluded that it would be a workaround for alerts not acting properly, which they _should_ be. I'd like for future devs to be able to use the `Alert` component wherever and have it work as expected. I'm happy to talk through it more (maybe at the next a11y jam?).

Reply:
That sounds great, let’s do it!